packages/one-app-dev-bundler/esbuild/utils/load-styles.js

```diff
@@ -50,7 +50,8 @@ const loadStyles = async ({
   let cssModulesJSON = {};
   const purgecssConfig = getModulesBundlerConfig('purgecss') || {};
   const result = await postcss([
-    (process.env.NODE_ENV === 'production' && !purgecssConfig.disabled) && purgecss(purgecssConfig),
+    // purgecss will not execute unless `disabled is set to `false` (opt-in by default)
+    (process.env.NODE_ENV === 'production' && purgecssConfig.disabled === false) && purgecss(purgecssConfig),
```

Comment:
Arguably, it might be worth removing the check here for `NODE_ENV` to get a consistent experience between environments, particularly because `purgecss` is opt-in now.

Reply:
Agreed